src/Mod/TechDraw/App/DrawViewDimension.h

```diff
@@ -105,8 +105,8 @@ class TechDrawExport DrawViewDimension : public TechDraw::DrawView
     App::PropertyString            FormatSpecOverTolerance;
     App::PropertyBool              Arbitrary;
     App::PropertyBool              ArbitraryTolerances;
-    App::PropertyFloat             OverTolerance;
-    App::PropertyFloat             UnderTolerance;
+    App::PropertyDistance          OverTolerance;

```

Comment:
See https://forum.freecadweb.org/viewtopic.php?f=35&t=52630&start=20#p453639

Reply:
Yes, indeed!  I'll try to use PropertyQuantity and make a new commit instead, thanks!